Backend/dev/migrations-read-only/dynamic-offer-driver-app/bap_metadata.sql

```diff
@@ -5,4 +5,21 @@ ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD COLUMN logo_url text NOT NUL
 ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD COLUMN name text NOT NULL;
 ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD COLUMN created_at timestamp with time zone NOT NULL default CURRENT_TIMESTAMP;
 ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD COLUMN updated_at timestamp with time zone NOT NULL default CURRENT_TIMESTAMP;
-ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD PRIMARY KEY ( id);
\ No newline at end of file
+ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD PRIMARY KEY ( id);
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_driver_offer_bpp.bap_metadata ADD COLUMN subscriber_id text NOT NULL;
```

Comment:
here `id` is `bap_subscriber_id` only, no need to add this new column.
Also merge drop not null queries by removing this change and regenrating.